db/migrate/20170923230214_create_likes.rb

```diff
@@ -1,11 +1,12 @@
 class CreateLikes < ActiveRecord::Migration[5.1]
   def change
-    create_table :likes do |t|
-      t.integer :user_id, null: false
-      t.integer :content_id, null: false
-      t.string :content_type, null: false
+    create_table :likes, id: false do |t|
```

Comment:
Why id false? If this is to enforce one like per user/content, why not just do uniqueness validations instead?

Reply:
Why do we need an id? It's essentially just a join table.